README.md

```diff
@@ -7,56 +7,68 @@ This is the [Sanity](https://sanity.io/) Studio for editing content on Regen Reg
 
 ```sh
-yarn deploy-staging
+yarn deploy-graphql:staging
 ```
 
-The GraphQL playground for staging dataset is available from https://jm12rn9t.api.sanity.io/v1/graphql/staging/default
-There's another playground for production dataset at https://jm12rn9t.api.sanity.io/v1/graphql/production/default
+The GraphQL playground for staging dataset is available from <https://jm12rn9t.api.sanity.io/v1/graphql/staging/default>.
+There's another playground for production dataset at <https://jm12rn9t.api.sanity.io/v1/graphql/production/default>.
 
-These can be used for querying, but not mutations (see sanity's [docs](https://www.sanity.io/docs/migrating-data) on migrations)
+These can be used for querying, but not mutations (see sanity's [docs](https://www.sanity.io/docs/migrating-data) on migrations).
 
 ### Deploying to production
 
-When merging a PR, we have a Github workflow defined to deploy the latest changes to the [main studio](https://regen.sanity.studio/desk) automatically.
+When merging a PR, we have a Github workflow defined to deploy the latest changes to the [main studio](https://regen.sanity.studio/desk) and the production GraphQL API.
+
+After merging a PR into the `main` branch, you might also want to import certain documents or parts of documents that you have created/updated during development from the staging dataset into the production dataset.
+There are a few scripts set up to do that automatically so you don't have to copy the content again manually:
 
-After merging a PR into the `main` branch, make sure to run:
+#### To create new documents
 
 ```sh
-yarn deploy-production
+yarn create-documents documentTypeName1 documentTypeName2
 ```
 
-This will update the production GraphQL API with latest schema, export the staging dataset and import it into the production dataset, so that data that you've added while in development on the staging dataset can be automatically copied into the production dataset without the need to do it manually. Later on, we can add that to Github workflows too so that it's all handled by CI.
+This might be useful if you created brand new documents or re-arranged existing ones extensively and you don't care about the existing documents (since it's using [`createOrReplace` mutation](https://www.sanity.io/docs/http-mutations#95bb692d7fb0)).
+This also traverses through all referenced documents, find all assets used by any of these documents in the staging dataset, then add them to the production dataset. But the depth of graph traversal is limited to 1 for now so if you want to add documents with deeper references, they will have to be specified manually in the document types list as first items.
+
+#### To patch existing documents
+
+```sh
+yarn patch-document documentTypeName fieldName
+```
+
+This might be useful if you add a new field to a document type.
+Under the hood, this uses [`set` within a `patch` mutation](https://www.sanity.io/docs/http-patches#6TPENSW3).
```

Comment:
If we feel the need to, we might want to add support for other kind of patch operations in the future, like unset or insert for instance